awx/ui_next/src/screens/Job/Job.jsx

```diff
@@ -14,15 +14,15 @@ import { CaretLeftIcon } from '@patternfly/react-icons';
 import { Card, PageSection } from '@patternfly/react-core';
 import { JobsAPI } from '../../api';
 import ContentError from '../../components/ContentError';
+import ContentLoading from '../../components/ContentLoading';
 import RoutedTabs from '../../components/RoutedTabs';
 import useRequest from '../../util/useRequest';
 import JobDetail from './JobDetail';
 import JobOutput from './JobOutput';
 import { WorkflowOutput } from './WorkflowOutput';
 import useWsJob from './useWsJob';
-import { JOB_TYPE_URL_SEGMENTS } from '../../constants';
 
-function Job({ i18n, lookup, setBreadcrumb }) {
```

Comment:
I couldn't find where we were passing the `lookup` prop and it was always `undefined` so I removed it.